### DISCUSSION TITLE
Need help setting up AMP on unraid

### DISCUSSION THREAD

**Discussion by phigiaon** [MitchTalmadge/AMP-dockerized#135] [Q&A]
Hey all I need some help setting up AMP in Unraid, I have custom MAC Address like in the guide, but got some questions regarding some settings and networking. Do I need to run AMP on Host network since the IP are just 0.0.0.0. Thanks!
![unknown-6](https://user-images.githubusercontent.com/96665459/185955212-6cf22905-1c14-4ec0-b475-bd5759e0e691.png)
![unknown-5](https://user-images.githubusercontent.com/96665459/185955221-fdd52855-b5e8-4cb7-a4b8-3053cfd5f688.png)
![unknown-4](https://user-images.githubusercontent.com/96665459/185956161-518781bd-1592-49e1-83f7-5cd1d3b051b3.png)

**Comment by VincedoubleO** [comment]
Just use default 0.0.0.0 uses default IP address of the machine 192.168.xx.xx:8080 port that amp ads uses or if you tailscale use the tailscale ip of the machine:8080
